7 punk mission statement copy

snuggs opened the pull request:
Fixes #7

https://github.com/NYCrb/devpunks/projects/1#card-1369683

snuggs commented:
@btakita The revolution begins.

snuggs commented on public/index.html:
```diff
@@ -3,14 +3,23 @@
-
-<blockqoute>
-</blockquote>
+<h2>the first technology show from <marquee style='color:red'>?????</marquee> to super developer in 15 minutes.</h2>
```
@melaniegershman suggestions for the `???` please ;-) You know better than I do.

melaniegershman commented:
@snuggs thoughts?

snuggs commented on public/index.html:
```diff
@@ -1,17 +1,42 @@
 
+  <section id='mission'>
+
+  <p>The devPunk subculture, includes a diverse array of ideologies, fashions and forms of expression, including visual art, literature, and film. The subculture is largely characterized by <strong><u>anti-establishment</u> views and the promotion of <u>individual freedom</u></strong>. The devPunk subculture is centered on a loud, aggressive genre of developers. It is usually played by small team ...
```
@melaniegershman what about "A developer and their mentor" removes this junior/senior bullshit. Death to it truth be told!

snuggs commented:
Seems like we have a merge conflict @melaniegershman. *warning*I'll rebase and force push. So any local branch you have consider it deprecated.

mcantor commented on public/index.html:
```diff
@@ -11,7 +11,31 @@
   <img src='/quote.png' alt='Dev Punks'>
 
+  <section id='mission'>
+  <p>The devPunk subculture, includes a diverse array of ideologies, fashions and forms of expression, including visual art, literature, and film. The subculture is largely characterized by <strong><u>anti-establishment</u> views and the promotion of <u>individual freedom</u></strong>. The devPunk subculture is centered on a loud, aggressive genre of developers. It is usually played by small team ...
```
No need to specify "subculture" here imo. We can just describe devPunk directly. Really hits the ear harder that way: "devPunk is usually played by small teams".

snuggs replied:
OHH I see what you did there @mcantor. It's in a way making a noun a verb and vice versa. _(see Google)_ #FunFact: They often call themselves _Googles_ on campus.

mcantor commented on public/index.html:
```diff
@@ -11,7 +11,31 @@
+  <section id='community'>
+    <header>Community</header>
+
+    <p>devPunks can come from any and all walks of life and economic classes. Compared to some alternative cultures, a devPunk is much closer to being gender equalist in terms of its ideology.
```
~~a devPunk~~ ➠ devPunk

mcantor commented on public/index.html:
```diff
@@ -11,7 +11,31 @@
+
+    <p>devPunks can come from any and all walks of life and economic classes. Compared to some alternative cultures, a devPunk is much closer to being gender equalist in terms of its ideology.
+
+    <p>One part of devPunks is creating explicitly outward identities of mentality. Everything that was normally supposed to be hidden was brought to the front, both literally and figuratively.
```
~~devPunks~~ ➠ devPunk

mcantor commented on public/index.html:
```diff
@@ -11,7 +11,31 @@
   <section id='media'>
+    <header>DevPunk Vodcast</header>
+
+    <p>DevPunks often follows a DIY policy of filming the footage first, without permission, and also posting it without permission, but will generally take down material if asked. Founder Joly MacFie argues that this serves an important archival purpose.
```
~~follows~~ ➠ follow

snuggs commented:
@mcantor https://github.com/NYCrb/devpunks/pull/8/commits/7d52a023b66aa9c744aff879440c008069818d90

snuggs commented:
@mcantor https://github.com/NYCrb/devpunks/pull/8/commits/999f19903f976c07684260170a21b4a636f5f660

changemewtf committed:
Clean up and refactor.

- Removed some unused images in the public folder
- Extracted copy from index.html into yaml files in data/
- Build homepage from index.haml instead of statically
- Condense Rack logic to events.rb, homepage.rb, and config.ru

I tried to use map "/" { run HomePage.new } to route just root to the
homepage, but that makes everything starting with "/" get served by
the HomePage app. So I used a lambda in place to test the request path
and route to HomePage if and only if the PATH_INFO is exactly "/".
Everything else will be passed through to NotFound.

changemewtf committed:
Refactor EasyData transformations.

changemewtf committed:
Simplify nested hash boolean logic.
